docs/page-tracking.md

```diff
@@ -56,16 +56,18 @@ The examples of creating full and incremental backups using the `--page-tracking
 After enabling the functionality, the next incremental backup finds changed
 pages using page tracking.
 
-The first full backup using page tracking, Percona XtraBackup may have a delay. The following is an example of the message:
+### Limitations
 
-??? example "Expected output"
+1. When creating the first full backup using page tracking, Percona XtraBackup may have a delay. The following is an example of the message:
 
     ```{.text .no-copy}
     xtrabackup: pagetracking: Sleeping for 1 second, waiting for checkpoint lsn 17852922 /
     to reach to page tracking start lsn 21353759
     ```
 
-Enable page tracking before creating the first backup to avoid this delay. This method ensures that the page tracking log sequence number (LSN) is higher than the checkpoint LSN of the server.
+    Enable page tracking before creating the first backup to avoid this delay. This method ensures that the page tracking log sequence number (LSN) is higher than the checkpoint LSN of the server.
+
+2. If the user runs LOCK=EXCLUSIVE and ALGORITHM=INPLACE DDL operations, Percona XtraBackup may not take a consistent backup of the MySQL server using page tracking. The corruption may happen because running the DDL operations is not added to the redo log during  `--prepare`, so Percona XtraBackup cannot detect that all modified pages are flushed to the disk.
```

Comment:
If the user runs LOCK=EXCLUSIVE and ALGORITHM=INPLACE DDL operations, Percona XtraBackup may not take a consistent backup of the Oracle MySQL server using page tracking. This is because of a server bug https://bugs.mysql.com/bug.php?id=106163. Oracle MySQL Server doesn't write the operations to redo log on such DDLs. Percona XtraBacup on a Percona Server for MySQL is not affected by the above bug, as it fixed in Percona Server for MySQL

XtraBackup cannot detect the inconsistency during the backup phase and can be detected only during the --prepare.

Reply:
updated